resources/sass/app.scss

```diff
@@ -13,7 +12,7 @@ html {
 }
 
 body {
```

Comment:
Documenting here too the following known issue: in staging, a font-size of 0.875em is coming from body and overriding all text on both the Mismatch finder and the Mismatch store. So far, I've been unable to locate where that value is coming from, since value replacement and imports look fine 🤔

Reply:
I can't see that 0.875em override in production anymore! 💯